LandSandBoat/server
# Ceizak NM - Unfettered Twitherym - behavior, hp, mp, mob skills.

**_I affirm:_**

## What does this pull request do?

This PR introduces mechanics to the Unfettered Twitherym NM, adding functionality for its mob skills, spells, HP/MP/Job/ItemLevel attributes and more. Specifically, after the mob performs a mob skill (at 74.49% HP), it becomes resistant to physical, magical, breath, and ranged damage. The only way to remove its damage resistance is by performing a skillchain with the correct magic affinity (e.g., fire, water, etc.).

Fire, Wind, Lightning, and Light work with Light.
Other elements work with their respective affinities (e.g., Fire skillchains for fire-based mobs).



## Steps to test these changes

Pop the NM using a Celadon Ygrette I at the appropriate location (J-7).
Engage the Unfettered Twitherym and observe its behavior as its HP reaches 74%, 49% and 24%.
Test the damage resistance by attacking with different types of damage (physical, magical, breath, ranged).
Perform a skillchain with the correct magic affinity (e.g., fire for fire-based, light for light-based).
Verify that the damage resistance is removed after the correct skillchain is executed.
